homework_11/codewars.py

```diff
@@ -0,0 +1,105 @@
+"""DESCRIPTION:
+Take an array and remove every second element from the array. Always keep the first element and start removing with the
+next element.
+
+Example:
+["Keep", "Remove", "Keep", "Remove", "Keep", ...] --> ["Keep", "Keep", "Keep", ...]"""
+
+
+def remove_every_other(my_list):
+    result = []
+    result.append(my_list[0])
+    for i in range(2, len(my_list), 2):
+        result.append(my_list[i])
+    return result
+
+
+"""Your goal in this kata is to implement a difference function, which subtracts one list from another and returns the
+ result.
+
+It should remove all values from list a, which are present in list b keeping their order.
+
+array_diff([1,2],[1]) == [2]
+If a value is present in b, all of its occurrences must be removed from the other:
+
+array_diff([1,2,2,2,3],[2]) == [1,3]"""
+
+
+def array_diff(a, b):
```

Comment:
this solution could be low performace, try to use casting to types, cast to set or tupes and try to find function for subtracts